### ISSUE TITLE
UNet Deprecation

### ISSUE THREAD

**Issue by jholland918** [alvyxaz/barebones-masterserver#225]
Now that Unity has deprecated UNet (https://blogs.unity3d.com/2018/08/02/evolving-multiplayer-games-beyond-unet/), are there any plans on switching out the UNet dependencies to another transport layer? 

I suppose just upgrading to whatever libraries Unity comes out with next might also be an option.

**Comment by hyakugei** [comment]
I have an interest in getting it working with Mirror (https://github.com/vis2k/Mirror), which is a replacement based on the old UNET HLAPI. I can see that there is already a branch somebody has created for this, and so am reaching out to them about doing the work to get it working with Mirror.

**Comment by jholland918** [comment]
I tried switching out the UNet dep with vis2k/Mirror a few weeks ago. There were a couple config classes that Mirror didn't have so I stopped because I'm no expert in networking. I suppose another option would be to look into another open source project like RevenantX/LiteNetLib.

**Comment by hyakugei** [comment]
Yeah, in smashing my head against this for 3 hours last night, i think it will be easier to actually do something else. Am now doing a Proof-of-Concept in Unity using a HTTP server to launch processes(games) via GET requests. So far so good.

**Comment by JordanDargaville** [comment]
Did anyone get Mirror working with barebones-masterserver? I've been trying to manually swap it out, but there seems to be a lot missing.

**Comment by Knightluozichu** [comment]
can use multiplayer HLAPI
